docs/client/configure-kn.md

```diff
@@ -0,0 +1,32 @@
+---
+title: "Customizing kn"
+weight: 03
+type: "docs"
+---
+
+You can customize your `kn` CLI setup by creating a `config.yaml` configuration file. The CLI will look for this file under the home directory of the user at `$HOME/.config/kn/config.yaml`, however you can create this file anywhere and use the `--config` flag to specify its path.
+
+## Example configuration file
+
+```yaml
+lookupPluginsInPath: true
+pluginsdir: ~/.config/kn/plugins
+sink:
+- prefix: myprefix
+  group: eventing.knative.dev
+  version: v1alpha1
+  resource: brokers
```

Comment:
```suggestion
plugins:
  pathLookup: true
  directory: ~/.config/kn/plugins
eventing:
  sink:
  - prefix: svc
    group: core
    version: v1
    resource: services
```

Apologies, that the original documentation is not up to date (going to send a PR to fix https://github.com/knative/client/tree/main/docs#kn-config).

(also I updated the sample mapping to a more realistic example, as this could be used directly to map Kubernetes Services to a `svc:` prefixes (while Knative Services are mapped to `ksvc:` already by default)

This means, we have introduced a grouping in the configuration on the first level (`plugins` and `eventing`), this might deserve separate sub-sections maybe in the documentation ?

Reply:
Made some updates. I think giving them separate sections for now might be overkill tbh until we get to adding more complex docs about configuration for `kn`. IMO for now it's enough to explain them in one example that can be iterated later.